### PR TITLE
Lucashh/refresh

### PR DESCRIPTION
**Pull request by lucashuet93**
- updates Azure Function to use .NET 8 and isolated worker model
- updates Front Door from Classic to Standard/Premium
- brings all resources up to date with current schemas
- updates documentation

### CODE REVIEW

File: src/inventory-api/inventory-api/Functions/GetProducts.cs [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
 using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Logging;
 using Inventory.Models;
-using System.Collections.Generic;
-using System;
 
-namespace Inventory.Functions
+namespace inventory_api.Functions
```
Comment (ckittel) [inline src/inventory-api/inventory-api/Functions/GetProducts.cs:7]: Was this namespace change expected?  I noticed some are still named `Inventory`, such as `Inventory.Models`

File: terraform/main.tf [hunk 2]
```diff
@@ -243,8 +223,9 @@ resource "azurerm_key_vault" "keyvault" {
     tenant_id = data.azurerm_client_config.current.tenant_id
     object_id = data.azurerm_client_config.current.object_id
     secret_permissions = [
-      "get",
-      "set"
+      "Get",
+      "Set",
```
Comment (ckittel) [inline terraform/main.tf:227]: Why was "Set" added?

**Review by ckittel — APPROVED** [review]
Looks good!  I only have a couple questions, but you're welcome to merge, after you review and make a decision on them -- no matter the answers.

File: src/inventory-api/inventory-api/Functions/GetProducts.cs (hunk 1 above)
Comment (lucashuet93) [reply src/inventory-api/inventory-api/Functions/GetProducts.cs:7]: Ah good catch, missed this. Fixed!

**Pull request merged**